docs/source/en/performance.mdx

```diff
@@ -46,31 +52,31 @@ In some cases training on a single GPU is still too slow or won't fit the large
 
 ### TPU
 
-_Coming soon_
+[_Coming soon_](perf_train_tpu)
 
 ### Specialized Hardware
 
-_Coming soon_
+[_Coming soon_](perf_train_special)
 
 ## Inference
 
 Efficient inference with large models in a production environment can be as challenging as training them. In the following sections we go through the steps to run inference on CPU and single/multi-GPU setups.
 
 ### CPU
 
-[Go to CPU inference section](perf_infer_cpu.mdx)
+[Go to CPU inference section](perf_infer_cpu)
 
 ### Single GPU
 
-_Coming soon_
+[_Coming soon_](perf_infer_gpu_one)
 
 ### Multi-GPU
 
-_Coming soon_
+[_Coming soon_](perf_infer_gpu_many)
```

Comment:
Not sure I understand why you want to remove "coming soon" from the two infer GPU sections and not the others? Because they have a link to the train on GPU section? I feel like a hint that these are at least WIP on the overview page would be useful not to be disappointed when clicking on them and not finding much information, yet.

Reply:
because nobody is going to click on it if it says Coming Soon, and once they click on it they have an actual lead to useful information from within the doc.

The TPU doc leads nowhere, hence it's fine to leave it as non-existing/placeholder.

Reply:
Ok, I makes sense now.